Homework1.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Scanner;
+
+/**
+ * @autor Created by mariavasilenko on 12/3/18.
+ */
+public class Homework1 {
+
+    public static void main(String[] args) {
+        Scanner s = new Scanner(System.in);
+        System.out.println("My name is " + getName(s) + " and I  " + getAge(s) + " years old");
+    }
+
+    private static int getAge(Scanner sc) {
+        try {
+            System.out.println("Enter your age");
+            String string2 = sc.nextLine();
```

Comment:
We need to use name of variable more understandable as you did below - String name = scanner.nextLine();